### PR TITLE
K8SPXC-569 update unsafe configuration option

### PR DESCRIPTION
**Pull request by fiowro**
[K8SPXC-569](https://jira.percona.com/browse/K8SPXC-569) [Powered by Pull Request Badge](https://pullrequestbadge.com/?utm_medium=github&utm_source=percona&utm_campaign=badge_info)

### CODE REVIEW

File: source/operator.rst [hunk 1]
```diff
@@ -832,7 +834,8 @@ configuration options for the HAProxy service.
 | **Example**     | ``3``                                                                                     |
 +-----------------+-------------------------------------------------------------------------------------------+
 | **Description** | The number of the HAProxy Pods `to provide load balancing                                 |
-|                 | <https://www.percona.com/doc/percona-xtradb-cluster/8.0/howtos/haproxy.html>`__           |
+|                 | <https://www.percona.com/doc/percona-xtradb-cluster/8.0/howtos/haproxy.html>`__. It should|
+|                 | be more than 1 unless the ``spec.allowUnsafeConfigurations`` key is set to true.          |
```
Comment (delgod) [inline source/operator.rst:838]:
```suggestion
|                 | be more than 2 unless the ``spec.allowUnsafeConfigurations`` key is set to true.          |
```

File: source/operator.rst [hunk 2]
```diff
@@ -832,7 +834,8 @@ configuration options for the HAProxy service.
 | **Example**     | ``3``                                                                                     |
```
Comment (delgod) [inline source/operator.rst:834]: new default is `2`
```suggestion
| **Example**     | ``2``                                                                                     |
```

File: source/operator.rst [hunk 3]
```diff
@@ -1364,11 +1367,11 @@ configuration options for the ProxySQL daemon.
...
 | **Value**       | int                                                                                       |
 +-----------------+-------------------------------------------------------------------------------------------+
-| **Example**     | ``1``                                                                                     |
+| **Example**     | ``3``                                                                                     |
 +-----------------+-------------------------------------------------------------------------------------------+
 | **Description** | The number of the ProxySQL daemons `to provide load balancing                             |
-|                 | <https://www.percona.com/doc/percona-xtradb-cluster/5.7/howtos/proxysql.html>`__          |
-|                 | must be = 1 in current release                                                            |
+|                 | <https://www.percona.com/doc/percona-xtradb-cluster/5.7/howtos/proxysql.html>`__. It      |
+|                 | should be more than 1 unless the ``spec.allowUnsafeConfigurations`` key is set to true.   |
```
Comment (delgod) [inline source/operator.rst:1374]:
```suggestion
|                 | should be more than 2 unless the ``spec.allowUnsafeConfigurations`` key is set to true.   |
```

File: source/operator.rst [hunk 4]
```diff
@@ -1364,11 +1367,11 @@ configuration options for the ProxySQL daemon.
 +-----------------+-------------------------------------------------------------------------------------------+
 | **Value**       | int                                                                                       |
 +-----------------+-------------------------------------------------------------------------------------------+
-| **Example**     | ``1``                                                                                     |
+| **Example**     | ``3``                                                                                     |
```
Comment (delgod) [inline source/operator.rst:1370]: new default is `2`
```suggestion
| **Example**     | ``2``                                                                                     |
```

**Review by delgod — CHANGES_REQUESTED** [review]

**Review by spron-in — CHANGES_REQUESTED** [review]
Please fix the comments from Mykola

**Review by delgod — APPROVED** [review]

File: source/operator.rst [hunk 5]
```diff
@@ -187,8 +187,10 @@ configuration options for the Percona XtraDB Cluster.
 +-----------------+-------------------------------------------------------------------------------------------+
 | **Example**     | ``3``                                                                                     |
 +-----------------+-------------------------------------------------------------------------------------------+
-| **Description** | The size of the Percona XtraDB cluster must be >= 3 for                                   |
-|                 | `High Availability <https://www.percona.com/doc/percona-xtradb-cluster/5.7/intro.html>`_  |
+| **Description** | The size of the Percona XtraDB cluster must be >= 3                                       |
+|                 | `High Availability <https://www.percona.com/doc/percona-xtradb-cluster/5.7/intro.html>`_. |
+|                 | It also should not exceed 5 and should be an even number unless the                       |
+|                 | ``spec.allowUnsafeConfigurations`` key is set to true.                                    |
```
Comment (spron-in) [inline source/operator.rst:193]: lets simplify this. We allow only 3 or 5 if unsafe is false.

**Review by spron-in — CHANGES_REQUESTED** [review]

**Review by spron-in — APPROVED** [review]

**Pull request merged**